### PR TITLE
added Homework_2, added 18 new files

### PR DESCRIPTION
**Pull request by UniAleks**

### CODE REVIEW

File: Homework2/TaskFour.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import java.util.stream.IntStream;
+//написать класс который выводит в консоль cумму всех eлeментов массива возведенных в квадрат
+public class TaskFour {
+    public static void main(String[] args) {
+        int mas[] = {1, 3, 6, 9, 4, 5, 9, 11, 43, 76};
+        int square = 2;
+        int sum = IntStream.of(mas).sum();
+        System.out.println(Math.pow(sum,square));
```
Comment (YeaaaaH) [inline Homework2/TaskFour.java]: a2+b2+c2 != (a+b+c)2

File: Homework2/TaskNine.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Print array length");
+        int value = scanner.nextInt();
+        int[] mas = new int[value];
+        System.out.println(mas);
+        for (int i = 0; i < value; i++) {
+            mas[i]= scanner.nextInt();
+        }
+        System.out.println(mas);
```
Comment (YeaaaaH) [inline Homework2/TaskNine.java]: you don't need this line

File: Homework2/TaskNine.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import java.util.Scanner;
+//написать класс который выводит в консоль массив состоящий из двух елементов.
+//1й - кол-во  всех позитивных значений массива, 2й - сумма всех отрицательных. Массив вводится с консоли
+public class TaskNine {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Print array length");
+        int value = scanner.nextInt();
+        int[] mas = new int[value];
+        System.out.println(mas);
```
Comment (YeaaaaH) [inline Homework2/TaskNine.java]: you don't need this line

File: Homework2/TaskOne.java [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import java.util.stream.IntStream;
+//написать класс который выводит в консоль сумму всех абсолютных значений массива
+public class TaskOne {
+    public static void main(String[] args) {
+        int mas[] = {1, 3, 6, 9, 4, 5, 9, 11, 43, 76};
+        int sum = IntStream.of(mas).sum();
```
Comment (YeaaaaH) [inline Homework2/TaskOne.java:6]: this function is not returning sum of absolute values

File: Homework2/TaskSeven.java [hunk 5]
```diff
@@ -0,0 +1,14 @@
+//написать класс который выводит в консоль число преобразованное из строки
+public class TaskSeven {
+    public static void main(String[] args) {
+        String Str = "753";
```
Comment (YeaaaaH) [inline Homework2/TaskSeven.java]: variables should be in lowerCase according to Naming Conventions

File: Homework2/TaskThree.java [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import static java.lang.Math.abs;
+//написать класс который выводит в консоль сумму всех целых чисел от 1 до заданного значения, число всегда больше 0
+public class TaskThree {
+    public static void main(String[] args) {
+        double mas[] = {1, 7, 3.43, 2.8, 4.75, 3.33, 9.5, 21};
```
Comment (YeaaaaH) [inline Homework2/TaskThree.java:5]: you don't need this array, get element from input stream

**Review by YeaaaaH — CHANGES_REQUESTED** [review]
need to fix comments

File: Homework2/TaskNine.java (hunk 2 above)
Comment (UniAleks) [reply Homework2/TaskNine.java]: fixed

**Pull request merged**